chalicelib/checks/helpers/wfr_utils.py

```diff
@@ -206,7 +206,7 @@
     'sci-Hi-C': [''],
     'MC-3C': [''],
     'MC-Hi-C': [''],
-    'Hi-ChIP': [''],
+    'Hi-ChIP': ["HiC_Pipeline_0.2.6", "HiC_Pipeline_0.2.7"],
```

Comment:
There is no need to have version 0.2.6 listed - you should remove that version from the list.  Should also add the same comment as there is on Plac-seq i.e. # Preliminary - Released to network # NO-NORM.